internal/repository/halls/hall.go

```diff
@@ -0,0 +1,55 @@
+package halldb
+
+import (
+	"database/sql"
+)
+
+type Repository struct {
+	DB *sql.DB
+}
+
+type Resource struct {
+	ID    int  `json:"ID"`
+	VIP   bool `json:"VIP"`
+	Seats int  `json:"seats"`
+}
+
+//Create new Hall in DB
+func (r *Repository) Create(hall Resource) error {
+	insertHall := `insert into halls("vip","seats") values($1,$2)`
+	_, err := r.DB.Exec(insertHall, hall.VIP, hall.Seats)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+//Retrieve Hall from DB
```

Comment:
```suggestion
// Retrieve Hall from DB
```
https://go.dev/doc/effective_go